package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@tigrisdata/core",
-	"version": "1.0.0-beta.11",
+	"version": "1.0.0-beta.12",
```

Comment:
this should reflect the current version. The release plugin will bump this before publishing.

Reply:
I see.